addon/services/construction-project.js

```diff
@@ -72,12 +71,6 @@ export default class ConstructionProjectService extends GwrService {
     return this.createAndCache(xml);
   }
 
-  async addDefaultWork(projectId) {
-    const buildingWork = new BuildingWork();
-    buildingWork.kindOfWork = 6001;
-    return await this.addWorkToProject(projectId, buildingWork);
-  }
```

Comment:
We no longer need addDefaultWork anywhere?

Reply:
Check for Sonderbau, whether no work is necessary

Reply:
I tested it again and for sonderbau there is no work necessary

Reply:
As discussed, added work to all types of constructions